### PR TITLE
fix: escape list for Query/Builder::toRawSql

### PR DESCRIPTION
**Pull request by taka-oyama**
Spanner bindings can also include list values.

### CODE REVIEW

File: tests/ConnectionTest.php [hunk 1]
```diff
@@ -482,4 +483,23 @@ protected function performRollBack($toLevel): void
...
+        self::assertSame('0', $conn->escape(0));
+        self::assertSame('-1', $conn->escape(-1));
+        self::assertSame('1.1', $conn->escape(1.1));
+        self::assertSame('"a"', $conn->escape('a'));
+        self::assertSame('r"""' . "\n" . '"""', $conn->escape("\n"));
+        self::assertSame('[]', $conn->escape([]));
+        self::assertSame('["a"]', $conn->escape(['a']));
+        self::assertSame('[false]', $conn->escape([false]));
+        self::assertSame('[1]', $conn->escape([1]));
+        self::assertSame('[1.1]', $conn->escape([1.1]));
```
Comment (halnique) [inline tests/ConnectionTest.php:503]:
```suggestion
        self::assertSame('[1.1]', $conn->escape([1.1]));
        self::assertSame('[1, [2, [3]]]', $conn->escape([1, [2, [3]]]));
```

Shouldn't multidimensional arrays also be included in the assertion to test the circular process?

**Review by halnique — COMMENTED** [review]
Also, please update `CHANGELOG.md`.

File: tests/ConnectionTest.php (hunk 1 above)
Comment (taka-oyama) [reply tests/ConnectionTest.php:503]: multi dimensional arrays are not supported.

https://cloud.google.com/spanner/docs/reference/standard-sql/data-types#declaring_an_array_type

Comment (halnique) [reply tests/ConnectionTest.php:503]: Oops... It seems to pass the test case for multidimensional arrays, so I guess we need to fix it so it doesn't pass?

Comment (taka-oyama) [reply tests/ConnectionTest.php:503]: That should be the expected behavior. This has been fixed.

File: src/Connection.php [hunk 2]
```diff
@@ -37,6 +37,8 @@
 use Illuminate\Contracts\Support\Arrayable;
 use Illuminate\Database\Connection as BaseConnection;
 use Illuminate\Database\QueryException;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Str;
```
Comment (halnique) [inline src/Connection.php:41]:
```suggestion
```

Probably forgot to erase it.

**Review by halnique — APPROVED** [review]

**Pull request merged**